src/constants.ts

```diff
@@ -29,6 +29,7 @@ if (!CONTEXT) throw new Error('missing REACT_APP_CONTEXT from environment')
 if (CONTEXT !== 'arkhn' && CONTEXT !== 'aphp' && CONTEXT !== 'fakedata') {
   throw new Error("REACT_APP_CONTEXT must be either 'aphp', 'fakedata' or 'arkhn")
 }
+export const FHIR_REQUEST_MAX_COUNT = 500
```

Comment:
mhh the max won't always be the same for all kinds of resources. I think it's best to keep this directly in the request for now